### DISCUSSION TITLE
conversion of  bayer raw images into jpeg images in libvips

### DISCUSSION THREAD

**Discussion by shameerasl** [libvips/libvips#2772] [Q&A]
Hi. I am again here to  accquire more information regarding libvips commands.  Can libvips convert raw files to jpeg files? Type of the raw file is bayer rggb. I am using libvips in windows. And your small reference also will be great help.Thanks in advance

**Comment by jcupitt** [comment]
Hi @shameerasl, this is best handled by a specialist library. I'd look at libraw:

https://www.libraw.org/

There's been some discussion about integrating this with libvips, but no one has volunteered yet.

https://github.com/libvips/libvips/discussions/2075#discussioncomment-2332258

**Comment by kleisauke** [comment]
A dedicated LibRaw loader is now available on the master branch (which will be part of the upcoming 8.18 release).
